arrow-optics/src/main/kotlin/arrow/optics/extensions/setk/at/SetKAt.kt

```diff
@@ -0,0 +1,48 @@
+package arrow.optics.extensions.setk.at
+
+import arrow.core.SetK
+import arrow.core.SetK.Companion
+import arrow.optics.PLens
+import arrow.optics.extensions.SetKAt
+import arrow.optics.setK
+import arrow.optics.typeclasses.At
+
+/**
+ * cached extension
+ */
+@PublishedApi()
+internal val at_singleton: At<SetK<Any?>, Any?, Boolean> = object : SetKAt<Any?> {}
+
+@JvmName("at")
+@Suppress(
+  "UNCHECKED_CAST",
+  "USELESS_CAST",
+  "EXTENSION_SHADOWED_BY_MEMBER",
+  "UNUSED_PARAMETER"
+)
+@Deprecated(
+  "@extension kinded projected functions are deprecated",
+  ReplaceWith(
+  "at(i)",
+  "arrow.core.at"
+  ),
+  DeprecationLevel.WARNING
+)
+fun <A, T> PLens<T, T, SetK<A>, SetK<A>>.at(i: A): PLens<T, T, Boolean, Boolean> =
+    arrow.optics.typeclasses.At.setK<A>().run {
+  this@at.at<T>(i) as arrow.optics.PLens<T, T, kotlin.Boolean, kotlin.Boolean>
+}
+
+@Suppress(
+  "UNCHECKED_CAST",
+  "NOTHING_TO_INLINE"
+)
+@Deprecated(
+  "@extension kinded projected functions are deprecated",
```

Comment:
☝️ 

Used the default message from other methods in this class. Let me know if there is a better choice.